addons/account/tests/test_account_move_entry.py

```diff
@@ -397,6 +397,15 @@ def test_journal_sequence_format(self):
             self.assertEqual(next_move_month.name, sequence_next_month)
             self.assertEqual(next_move_year.name, sequence_next_year)
 
+    def test_journal_next_sequence(self):
+        prefix = "TEST_ORDER/2016/"
+        self.test_move.name = f"{prefix}1"
+        for c in range(2, 25):
+            copy = self.test_move.copy()
+            copy.name = "/"
+            copy.post()
+            self.assertEqual(copy.name, f"{prefix}{c}")
```

Comment:
I'd like also to see post() called upon a list of records that don't share the same journal/type

Reply:
Added `test_journal_sequence_multiple_type`